src/lib/queries/json.ts

```diff
@@ -14,7 +14,7 @@ export class JsonCommonQuery implements CommonQuery {
 
 	private client: Client;
 	private get provider() {
-		return this.client.providers.get('json') as JsonProvider;
+		return (this.client.providers.get('json') ?? this.client.providers.get('cache')) as JsonProvider;
```

Comment:
This is what I meant:
```suggestion
		return this.client.providers.get(this.client.providers.default) as JsonProvider;
```

A safer alternative:
```suggestion
		return this.client.providers.get(this.client.providers.default === 'json' ? 'json' : 'cache') as JsonProvider;
```

Reply:
Close, it would be `this.client.providers.default.name`, if you can edit this I can commit it easily.